### PR TITLE
 Add fault injection test and update stress tests file

### PR DESCRIPTION
**Pull request by Ostrokrzew**
Add fault injection test: try load cache with core device already mounted;
/*1*/ "Start Intel CAS device in wb mode",
/*2*/ "Add core device with xfs filesystem and mount it",
/*3*/ "Crate file in /big directory and count it md5sum",
/*4*/ "Copy file to Intel CAS device",
/*5*/ "Stop CAS",
/*6*/ "Mount Core device",
/*7*/ "Try Start cache with 'load' option with mounted core device",
/*8*/ "Check md5sum for file in core device"

Update test after separating functions from this test to other files.
Add 'dut_config' and 'duts_in_use' to .gitignore.

Signed-off-by: Slawomir_Jankowski <slawomir.jankowski@intel.com>

### CODE REVIEW

File: test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_load_cache_with_mounted_core(cache_mode):
+    """
+        title: Fault injection test for starting cache with mounted core.
+        description: |
+          False positive test of the ability of the CAS to load cache
+          when core device is mounted.
+        pass_criteria:
+          - No system crash while load cache.
+          - Core device loads unsuccessfully.
```
Comment (mmichal10) [inline test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py]: 'Loading core device fails.

File: test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    with TestRun.step(f"Create test file in {mount_point} directory and count its md5 sum."):
+        file = fs_utils.create_test_file(test_file_path)
+        md5_before_load = file.md5sum()
+        size_before_load = file.size
+        permissions_before_load = file.permissions
+
+    with TestRun.step("Copy file to cache device and release core."):
+        dd = Dd().input(file).output(cache).count(1)
+        dd.run()
+        core.unmount()
```
Comment (mmichal10) [inline test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py]: I dont understand meaning of this step.

File: test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py (hunk 1 above)
Comment (Ostrokrzew) [reply test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py]: It's in test's scenario.

File: test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py (hunk 2 above)
Comment (Ostrokrzew) [reply test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py]: It's in test's scenario, check it.

File: test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py (hunk 1 above)
Comment (Ostrokrzew) [reply test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py]: fixed

File: test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py (hunk 2 above)
Comment (Ostrokrzew) [reply test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py]: done

**Pull request closed without merging**